client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,161 @@
+
+const JPOSummaryStep = React.createClass( {
+	propTypes: {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setJPOSummary: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	},
+
+	getFormattedPayload() {
+		let payload = this.props.signupProgress;
+		
+		let formattedPayload = {
+			siteTitle: payload[0].jpoSiteTitle.siteTitle,
+			siteDescription: payload[0].jpoSiteTitle.siteDescription,
+			businessPersonal: payload[1].jpoSiteType.businessPersonal,
+			genre: payload[1].jpoSiteType.genre,
+			businessName: payload[1].jpoSiteType.addressInfo.businessName,
+			businessAddress: payload[1].jpoSiteType.addressInfo.streetAddress,
+			businessCity: payload[1].jpoSiteType.addressInfo.city,
+			businessState: payload[1].jpoSiteType.addressInfo.state,
+			businessZipCode: payload[1].jpoSiteType.addressInfo.zipCode,
+			homepageFormat: payload[2].jpoHomepage,
+			addContactForm: payload[3].jpoContactForm
+		};
+
+		return formattedPayload;
+	},
+
+	completeOnboarding() {
+		// Get the payload and original JPC url
+		const payload = this.getFormattedPayload();
+		const jetpackConnectUrl = localStorage.getItem( 'jpoConnectUrl' );
+
+		// Flag the flow as complete for use in JPC
+		localStorage.setItem( 'jpoFlowComplete', '1' );
+
+		/**
+		 * Store the payload in localStorage for use after the connection
+		 * to Jetpack has been authorized
+		 */
+		localStorage.setItem( 'jpoPayload', JSON.stringify( payload ) );
+
+		// Redirect to the original JPC URL
+		window.location.href = jetpackConnectUrl;
+	},
+
+	goToStepOne() {
+		this.props.goToStep( 1 );
+	},
+
+	goToStepTwo() {
+		this.props.goToStep( 2 );
+	},
+
+	goToStepThree() {
+		this.props.goToStep( 3 );
+	},
+
+	goToStepFour() {
+		this.props.goToStep( 4 );
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 } icon="checkmark" />;
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header">Steps you've completed:</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{ 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle ? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[1].jpoSiteType ? <li>{ checkMark } { translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[2].jpoHomepage ? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[3].jpoContactForm ? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li> : null }
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Contact Us' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-flow-steps)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).